dbt_subprojects/hourly_spellbook/models/_metrics/fees/metrics_fees_index_daily.sql

```diff
@@ -0,0 +1,57 @@
+{{ config(
+        schema = 'metrics'
+        , alias = 'fees_index'
+        , materialized = 'incremental'
+        , file_format = 'delta'
+        , incremental_strategy = 'merge'
+        , unique_key = ['blockchain', 'block_date']
+        , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')]
+        )
+}}
+
+{% set baseline_date = '2018-01-01' %}
+
+with baseline as (
+    select
+        sum(gas_fees_usd) as baseline_gas_fees_usd -- sum is required due to blockchain being second unique key in source
+    from
+        {{ ref('metrics_gas_fees_daily') }}
+    where
+        block_date = date '{{ baseline_date }}'
+), blockchain_baseline as (
```

Comment:
I forgot to remove this from the prototype. We don't need the blockchain specific index calculations. This was something I was playing with early on because a blockchain may want their own version of the index that just calculates their adoption from their starting point. But for our purposes here we shouldn't include it because we're not using it.  We'd also need to calculate each chain's start date for their own index for it to work properly, but since we're not using it there's no need to go through all of that.